smact/lattice_parameters.py

```
@@ -7,8 +7,29 @@
+
+# Litharge (B10, PbO-type) empirical factors
+_LITHARGE_SUM_FACTOR = 1.31
+_LITHARGE_C_OVER_A = 1.26  # c/a from PbO data (http://www.mindat.org/min-2466.html)
+
+
+def _check_positive(values: list[float] | float, name: str = "radius") -> None:
+    """Raise ValueError if any radius value is not positive."""
+    if isinstance(values, (int, float)):
+        values = [values]
+    if any(v <= 0 for v in values):
+        raise ValueError(f"All {name} values must be positive, got {values}")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)